src/EventListener/ConditionallyRemoveVaryHeaderListener.php

```diff
@@ -48,7 +48,7 @@ public function __construct(array $routes, array $userIdentifierHeaders = ['Cook
      */
     public function onKernelResponse(ResponseEvent $event)
     {
-        if (HttpKernelInterface::MASTER_REQUEST != $event->getRequestType()) {
+        if (HttpKernelInterface::MAIN_REQUEST != $event->getRequestType()) {
```

Comment:
Since we're visiting this, we can also adjust it to use proper comparison type.
```suggestion
        if (HttpKernelInterface::MAIN_REQUEST !== $event->getRequestType()) {
```